README.md

```diff
@@ -80,30 +82,39 @@ Note: local PARA_ID is defaulted to 2000
 
 ### Launch the Relay Chain
 
+Please check out paritytech/polkadot’s [Build from Source](https://github.com/paritytech/polkadot/releases/) for full instructions, but first find the latest stable tag on [Polkadot Releases](https://github.com/paritytech/polkadot/releases/), for example, using `v0.9.31` as the `<latest_stable_tag>` below.
```

Comment:
For the easiest development experience, I suggest that folks use the relay chain version that corresponds to our most recent release. Can be found in the Cargo.toml. For example, https://github.com/OAK-Foundation/OAK-blockchain/blob/be8235419382a673168675a62df2ca0efd701a0b/runtime/neumann/Cargo.toml#L92 `v0.9.26` currently.

I expect that it will work fine in most cases even with the most recent stable release, but wanted to flag this as different than our current dev practice.

Reply:
Yes, good call. I’ll update the relay chain version.